.github/actions/build/action.yml

```diff
@@ -6,21 +6,15 @@ inputs:
   python-version:
     description: Python version
     required: true
-  build-mode:
```

Comment:
1st line of Commit message is tooling and overflows in GitHub.  Please shorten the 1st line (that becomes the header) and move some description text to 2nd line.